doc/paq-nvim.txt

```diff
@@ -99,14 +99,14 @@ imported as `paq`, the functions are:
 
 |paq.install|                                                    *paq.install*
                                                                *:PaqInstall*
-  Installs all packages listed on your configuration, if a package is already
+  Installs all packages listed in your configuration - if a package is already
```

Comment:
Question: Could this be a period instead?

Reply:
Yes, it could be a sentence on it's own, I'll switch it out! I was trying to not make too many structural changes with this PR.